core/lib/config/src/configs/eth_sender.rs

```diff
@@ -127,6 +128,10 @@ pub struct SenderConfig {
     /// special mode specifically for gateway migration to decrease number of non-executed batches
     #[serde(default = "SenderConfig::default_tx_aggregation_only_prove_and_execute")]
     pub tx_aggregation_only_prove_and_execute: bool,
+
+    /// Cap of time in mempool for price calculations
+    #[serde(default = "SenderConfig::default_time_in_mempool_cap")]
+    pub time_in_mempool_cap: u32,
```

Comment:
nit: include `_sec` in name

Reply:
it's not in seconds, but in l1 blocks, I've refactored all time_in_mempool references to time_in_mempool_in_l1_blocks to increase readability